code/modules/vore/eating/vorepanel_vr.dm

```diff
@@ -196,6 +196,12 @@
 		if(FALSE)
 			dat += "<a href='?src=\ref[src];toggledvor=1'>Toggle Devourable (Currently: OFF)</a>"
 
+	switch(user.sleeperable)
```

Comment:
Putting this in the vore panel basically means that they still have to look at the settings, it's okay to add this to modules/client/preferences and savefile. just look at 'window flashing' or 'lobby music' or some other yes/no choice for hints on how to implement it.